### ISSUE TITLE
Dark mode contrast issue (A11Y)

### ISSUE THREAD

**Issue by bwbroersma** [Logius-standaarden/publicatie#57]
See https://logius-standaarden.github.io/ReSpec-template/#normatieve-referenties

<img width="1708" height="420" alt="Image" src="https://github.com/user-attachments/assets/209220ed-7dd0-4163-9c43-843e383c0477" />

The problem is here:
https://github.com/Logius-standaarden/publicatie/blob/9feef9289c89336ce0caf3d45371cfa42b2780e2/fsc/logging/index.html#L51

In combination with dark mode, the result does not meet WCAG standards for accessible text.
